sdk/communication/azure-communication-chat/tests/_shared/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import unittest
```

Comment:
This file should be moved out of a `_shared` directory into just the `tests` directory. Otherwise I don't think it will be run.

Reply:
There is another set of test_communication_identifier_serializer UT in _shared folder. I validated it locally, it's included in pytest.